style.css

```diff
@@ -1,18 +1,17 @@
 #id-main-content {
```

Comment:
- [ ] You have used several `id` selectors in your CSS file. That is not the best practice way. Kindly replace them with class selectors. Don't forget to change the `id` attributes to `class` attributes in your HTML.

Reply:
This issue has been addressed ✔️